lang/main.json

```diff
@@ -440,8 +440,10 @@
         "hidePassword": "Hide password",
         "inviteTo": "Invite people to __conferenceName__",
         "invitedYouTo": "__userName__ has invited you to the __inviteURL__ conference",
+        "invitePeople": "Invite people",
         "locked": "This call is locked. New callers must have the link and enter the password to join.",
         "showPassword": "Show password",
+        "shareTheLink": "Share the link",
```

Comment:
I noticed this same copy being used for 4 different translation keys (one of them no longer referenced). Is there any guideline for this situation to prevent duplication? Is it an occurrence worth addressing? Should all the other usages be moved over to this one?

Reply:
Good catch. The translation file isn't currently structured in the best way to make it easy to avoid duplications. I guess the basic rule is to search for your string before adding one. I generally do that but have missed this one. I'll fix this.